### PR TITLE
Add a new positioning serial port receiver

### PR DESCRIPTION
**Pull request by nirvn**
Did someone say Bluetooth support for QField Windows through virtual serial port would be cool? Yep :)

More details on how that's done: https://junipersys.com/support/article/14876

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for f4a36a2e7e95c34b15f5f5b949deb0ad7aabbc0d: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3806-c40c9a3317e033b730600155fa5590d2a73886f8-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3806-c40c9a3317e033b730600155fa5590d2a73886f8-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3806-c40c9a3317e033b730600155fa5590d2a73886f8-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3806-c40c9a3317e033b730600155fa5590d2a73886f8-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3806-c40c9a3317e033b730600155fa5590d2a73886f8-arm64-android.apk)
- [all access arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3806-c40c9a3317e033b730600155fa5590d2a73886f8-arm-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3806-c40c9a3317e033b730600155fa5590d2a73886f8-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3806-c40c9a3317e033b730600155fa5590d2a73886f8-x86-android.apk)
</details>

**Comment by nirvn** [comment]
Tested with an old, old garmin device I found in my drawer (it should really be sent to a museum), we've got bluetooth through serial port working :)

**Comment by nirvn** [comment]
@m-kuhn , I think this is ready for review / approval, it has been tried and working on multiple machine.

File: .qt6/vcpkg.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "qtlocation",
     "qtcharts",
     "qtmultimedia",
+    "qtserialport",
```
Comment (m-kuhn) [inline .qt6/vcpkg.json:20]:
```suggestion
    {
      "name": "qtserialport",
      "platform": "windows"
    },
```

**Review by m-kuhn — APPROVED** [review]

File: vcpkg.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
     "qt5-location",
     "qt5-charts",
     "qt5-multimedia",
+    "qt5-serialport",
```
Comment (m-kuhn) [inline vcpkg.json:22]:
```suggestion
    {
      "name": "qtserialport",
      "platform": "windows"
    },
```

Comment (m-kuhn) [reply vcpkg.json:22]: I thought it's "only" a BT on Win hack -- but you convinced me already on backchannels

**Pull request merged**